src/main/java/pl/exsio/querydsl/entityql/type/CustomProjections.java

```diff
@@ -0,0 +1,54 @@
+package pl.exsio.querydsl.entityql.type;
+
+import com.google.common.collect.ImmutableList;
+import com.querydsl.core.types.Expression;
+import com.querydsl.core.types.Path;
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.NamingStrategy;
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.UnderscoreToCamelStrategy;
+
+import java.util.List;
+
+public final class CustomProjections {
```

Comment:
This name is not telling is much. Maybe MapProjections? StringMapProjections? SimpleMapProjections?

Reply:
Or maybe RuntimeProjections? If the feature you're adding should be used only when the schema is not known during compilation time, then it should be "Runtime"

Reply:
I don't think Projections has anything to do with runtime, renamed it SimpleMapProjections.